Card.css

```diff
@@ -0,0 +1,99 @@
+body {
+  font-family: "comic sans ms";
+  display: flex;
+  gap : 1.25rem;
+  flex-wrap: wrap;
+  
+}
+
+.card-container {position: relative;
+  display : flex;
+  
+  /* flex-wrap : wrap; */
```

Comment:
You may remove the commented out code and format it using an extension like prettier.